v2/openapi/fb-provider-api.yaml

```diff
@@ -607,6 +607,8 @@ components:
             - deposit-address-disabled
             - order-not-trading
             - idempotency-key-reuse
+            - deposit-address-creation-is-impossible
```

Comment:
maybe `deposit-address-creation-not-allowed`

Reply:
I prefer `deposit-address-creation-not-allowed` too.